.github/ISSUE_TEMPLATE/regular-issue.yaml

```diff
@@ -0,0 +1,37 @@
+name: Generic Issue
+description: Open a generic Issue for the OpenJS CPC
```

Comment:
```suggestion
description: Open a generic issue for the OpenJSF CPC
```

Reply:
> Use the full form of any trademarks (e.g., “OpenJS Foundation”) in the first reference in all documents of mass communication, including marketing collateral and web pages. You may then use any abbreviated or short form references (e.g. “OpenJSF”) within the same document where the full form has already been used.

Source: [OpenJS Foundation Trademark Policy, p. 2](https://trademark-policy.openjsf.org/)

Reply:
Please don't edit someone's comment unless there's a reasonable reason (safety, CoC, etc.) to do so. It makes it really confusing to follow what's going on. Thanks!